### DISCUSSION TITLE
Ideasguying Thread

### DISCUSSION THREAD

**Discussion by MarkSuckerberg** [Whitesands13/Whitesands#646] [Ideasguy]
Post ideas that don't really need their own thread here.

**Comment by Superyodeler** [comment]
I think it would be a good idea to have an ideasguy thread.

**Comment by Epsilos** [comment]
force the vote window to open when a vote starts so that people who complain about kilo or crew transfers have to go out of their way to disenfranchise themselves

**Comment by Doctor-Derp** [comment]
**Pretty Alert Consoles.** 

Orange Stripe - Power Alarm

Red and Yellow Stripes - Fire Alarm

Blue background - Nominal Atmospherics.
Yellow background - Warning.
Red background - Breach/Atmosphere alarm.

![image](https://user-images.githubusercontent.com/40336142/107061966-c444b000-67a6-11eb-8cfe-0a046d55ab7e.png)

**Comment by Doctor-Derp** [comment]
**Biohazard disposal crate/trashbags/incinerator for medical.**

People don't bother putting organs in the fridge, even if there is one, after surgery/using the autodoc.

If there's no fridge/someone butchers a monkey, they're just going to throw rotting organs down to the delivery room.

**Comment by Doctor-Derp** [comment]
**Bigger pill bottles.**

Current pill bottles can only hold 7 pills, what kinda of fuckin' pills are those?

**Reply by Maldaris** [reply]
> **Bigger pill bottles.**
> 
> Current pill bottles can only hold 7 pills, what kinda of fuckin' pills are those?

Also make them scoop all the pills on a tile like RPEDs
Same thing but with Folders too.

**Comment by Doctor-Derp** [comment]
Welded airlock mapping helper.

**Comment by Doctor-Derp** [comment]
**Pretty EVA suits.**

Why is it that people can't make good space suits?
Bay has some nice ones.
